tests/routes_tests/cbauthentication_tests/cb_password_reset.test.js

```diff
@@ -0,0 +1,43 @@
+const test = require('tape');
+const request = require('supertest');
+const app = require('../../../react-backend/app');
+const dbConnection = require('../../../react-backend/database/dbConnection');
+const rebuild = require('../../../react-backend/database/database_rebuild');
+
+test('POST api/cb/pwd/reset | token creation successful', async (t) => {
+  await rebuild();
+
+  const successPayload = { formEmail: 'findmyfroggy@frogfinders.com' };
+  request(app)
+    .post('/api/cb/pwd/reset')
+    .send(successPayload)
+    .expect(200)
+    .end(async (err) => {
+      t.notOk(err, err || 'Passes supertest expect criteria');
+      try {
+        const getTokenFromDb = await dbConnection.query(
+          "SELECT * FROM cbusiness WHERE (token IS NOT NULL) AND email = 'findmyfroggy@frogfinders.com'",
+        );
+        t.ok(getTokenFromDb.rows.length, 'route successfully created a token in db');
+        t.end();
+      } catch (error) {
+        t.fail('Test query broke');
```

Comment:
You can just do `t.end(error)`, that way we get the stacktrace in the test output directly. This way we'd have to put in a `console.log` to find out what happened.